.gitignore

```diff
@@ -66,5 +66,6 @@ typings/
 
 /.vscode
 /.idea
+.envrc
```

Comment:
this is something we can think generalization in one place

Reply:
I could add it to my own gitignore I guess :)

Reply:
I use it too, change is good. In terms of hack weeks, might want to think about how to maintain this and similar changes in an easier way